telos-backend/package.json

```diff
@@ -5,7 +5,8 @@
   "main": "app.js",
   "scripts": {
     "start": "node app.js",
-    "test": "jest"
+    "test": "jest",
+    "pretty": "prettier --write \"./**/*.{js,jsx,json}\""
```

Comment:
Did we also want a `lint` command to see changes but not make them? Or not have this functionality at all. Also we should add this to backend readme. But `pretty` command looks good to me.

Reply:
Good idea, I've just added it now.